pyproject.toml

```
@@ -120,7 +120,7 @@ filterwarnings = [
 testpaths = [
     "tests"
 ]
-timeout = 60
+# timeout = 60
```

```suggestion
timeout = 60
```

I am uncommenting this because this PR keeps taking up all the ci runner slots because of breakpoints left in committed code and other non-termination problems.